core/src/middleware/exceptionMiddleware.ts

```diff
@@ -21,7 +21,7 @@ export const ExceptionMiddleware = (options: ExceptionOptions): Middleware =>
 
       const result = {
         requestId: context.id,
-        message: err.toString(),
+        message: typeof err === "string" ? err : err instanceof Error ? err.toString() : JSON.stringify(err),
```

Comment:
nit: add some parenthesis to make it more readable

Reply:
fixed, thanks!